spec/ReadPreferenceOption.spec.js

```diff
@@ -0,0 +1,978 @@
+'use strict'
+
+const Parse = require('parse/node');
+const ReadPreference = require('mongodb').ReadPreference;
+const rp = require('request-promise');
+
+describe('Read preference option', () => {
+  it('should find in primary by default', (done) => {
+    const obj0 = new Parse.Object('MyObject');
+    obj0.set('boolKey', false);
+    const obj1 = new Parse.Object('MyObject');
+    obj1.set('boolKey', true);
+
+    Parse.Object.saveAll([obj0, obj1]).then(() => {
+      const hook = {
```

Comment:
@davimacedo why don't you just put a spy on the database.serverConfig.cursor and validate the spy later on (you can get all arguments etc...